cmd/ci-secret-bootstrap/main.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	coreapi "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"os"
+	"strings"
+
+	"github.com/openshift/ci-tools/pkg/bitwarden"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+type options struct {
+	dryRun         bool
+	kubeConfigPath string
+	configPath     string
+	bwUser         string
+	bwPasswordPath string
```

Comment:
This is not that related to the actual code, but BW credentials are a immensely sensitive secret: leaking it means leaking all secrets. Are we planning to run this tool just manually, or will we deploy it somewhere where we'll need to put also the credentials?

Reply:
Did not finalize this with Steve.
In my opinion, we should keep BW from our jobs and only run it locally/manually.
BW is too risky and hard to recover (James indicates that giving a user readonly access is not possible):
- rotation all is hard
- no git-blame or git-reset for BW